fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-0-SetUp.xml

```diff
@@ -46,11 +48,46 @@ This job performs the initial setup procedure for Sanity Check execution&#xd;
   <customWorkspace>$FIHEALTH_WORKSPACE</customWorkspace>
   <builders>
     <hudson.tasks.Shell>
-      <command># Workaround to spawn processes from build (https://wiki.jenkins-ci.org/display/JENKINS/Spawning+processes+from+build)
+      <command>#!/bin/sh
+
+# Workaround to spawn processes from build (https://wiki.jenkins-ci.org/display/JENKINS/Spawning+processes+from+build)
 BUILD_ID=dontKillMe
 
-# Execute &apos;setup&apos; script
-./$FIHEALTH_SANITY_PROJECT/resources/scripts/jenkins.sh setup
+
+# Very first run: download tests from GitHub repository
+if [ ! -d $WORKSPACE/.git ]; then
+	git clone --branch=master https://github.com/telefonicaid/fiware-health.git $WORKSPACE
+	cd $FIHEALTH_SANITY_PROJECT
+	printf 1&gt;&amp;2 &quot;\n[WARNING] Check configuration at $PWD/resources and re-run this job\n\n&quot;
+	exit 1
+fi
+
+# Subsequent runs: only upgrade tests if configuration backup exists
+cd $FIHEALTH_SANITY_PROJECT
+if [ -d resources.backup ]; then
+	git fetch
+	git pull origin master
+else
+	mkdir resources.backup
+	cp resources/*.conf resources.backup
+	cp resources/*.json resources.backup
+	printf &quot;\n[INFO] Copy of configuration files stored in $PWD/resources.backup\n\n&quot;
+fi
+
+# Check for configuration differences and restore backup if needed
+EXCLUDE_SUBDIRS=$(cd resources; for i in `ls -d */`; do echo -x ${i%/}; done)
+if ! diff -r $EXCLUDE_SUBDIRS resources resources.backup &gt;/dev/null; then
```

Comment:
At the moment, we can accept it but take into account that it is ok iff the keys in both are the same and only a change in values is registered. But what could we the solution if there is a change also in the keys.